internal/replication/utils.go

```diff
@@ -25,3 +27,41 @@ func raftEntryString(e raftpb.Entry) string {
 	return fmt.Sprintf("{Term:%d Index:%d Type:%v Data(len):%d Data(hash):%X}",
 		e.Term, e.Index, e.Type, len(e.Data), h.Sum64())
 }
+
+func ClassifyClusterReConfig(currentConfig, updatedConfig *types.ClusterConfig) (nodes bool, consensus bool, ca bool, admins bool) {
+	curNodes := currentConfig.GetNodes()
+	updNodes := updatedConfig.GetNodes()
+	if len(curNodes) != len(updNodes) {
+		nodes = true
+	} else {
+		for i, node := range curNodes {
```

Comment:
Do we keep nodes order in list? Can't it be same nodes, but in different order?

Reply:
Yes, good point.

Reply:
Done, for both nodes and admins.